process_pr.py

```diff
@@ -1595,15 +1600,16 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             break
 
     old_labels = set([x.name.encode("ascii", "ignore").decode() for x in issue.labels])
-    print("Stats:", backport_pr_num, extra_labels)
+    print("Stats:", backport_pr_num, extra_labels, state_labels)
     print("Old Labels:", sorted(old_labels))
     print("Compilation Warnings: ", comp_warnings)
     print("Singnatures: ", signatures)
     if "mtype" in extra_labels:
         extra_labels["mtype"] = list(set(extra_labels["mtype"]))
     if "type" in extra_labels:
         extra_labels["type"] = [extra_labels["type"][-1]]
-
+    if len(state_labels) != 0:
```

Comment:
there is no need to add this check. `extra_labels["mtype"].extend(state_labels.values())` should work for empty `state_labels` too